### PR TITLE
fix(yupResolver): multiple errors messages with same type

### PR DESCRIPTION
**Pull request by jorisre**
Related to #155 

I'll update the v1 as well

### CODE REVIEW

File: yup/src/yup.ts [hunk 1]
```diff
@@ -17,12 +17,17 @@ const parseErrorSchema = (
...
+      const messages = types && types[error.type!];
+
       previous[error.path!] = appendErrors(
         error.path!,
         validateAllFieldCriteria,
         previous,
         error.type!,
-        error.message,
+        // eslint-disable-next-line
+        // @ts-expect-error
```
Comment (jorisre) [inline yup/src/yup.ts:29]: Screenshot 2021-04-14 at 17 09 57

Comment (bluebill1049) [reply yup/src/yup.ts:29]: I think you can do a cast here.

**Review by bluebill1049 — APPROVED** [review]
Awesome! thanks @jorisre

File: yup/src/yup.ts (hunk 1 above)
Comment (jorisre) [reply yup/src/yup.ts]: Updated, you can review again

**Review by bluebill1049 — APPROVED** [review]
❤️ awesome.

**Pull request merged**